src/encoded/static/components/browse/MicroscopySearchView.js

```diff
@@ -0,0 +1,263 @@
+    "microscope.Serial": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "microscope.Model": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "microscope.Type": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "submitted_by.display_title": { "widthMap": { "sm": 100, "md": 100, "lg": 120 } },
+});
+export default class MicroscopySearchView extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.handleModalCancel = this.handleModalCancel.bind(this);
+        this.handleConfirm = _.throttle(this.handleConfirm.bind(this), 3000);
+        this.handleMicroscopeNameChange = this.handleMicroscopeNameChange.bind(this);
+
+        this.state = {
+            show: false,
+            tier: 1,
+            validationTier: 1,
+            microscopeName: '',
+            cloneLoading: false
+        };
+
+        this.searchResultTableRef = React.createRef();
+    }
+    leftButtonDropDownButton() {
+        const { show, tier, validationTier, microscopeName,cloneLoading } = this.state;
+        const validationTierOptions=_.range(1,tier+1);
+        return (
+            <>
+                <DropdownButton id="search-item-type-selector" onSelect={(eventKey, evt) => { this.onChangeCreateTier(eventKey); }}
+                    title={'Crete New Configuration'} >
+                    <DropdownItem eventKey="1" data-key="1" type="number" >
+                        Tier 1
+                    </DropdownItem>
+                    <DropdownItem eventKey="2" data-key="2" type="number" >
+                        Tier 2
+                    </DropdownItem>
+                    <DropdownItem eventKey="3" data-key="3" type="number" >
+                        Tier 3
+                    </DropdownItem>
+                    <DropdownItem eventKey="4" data-key="4" type="number" >
+                        Tier 4
+                    </DropdownItem>
+                    <DropdownItem eventKey="5" data-key="5" type="number" >
+                        Tier 5
+                    </DropdownItem>
+                </DropdownButton>
+
+
+                <Modal show={show} >
+                    <Modal.Header>
+                        <Modal.Title>New Microscope - Tier {tier} </Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        <div>
+                            <label>Microscope Name</label>
+                        </div>
+                        <div>
+                            <input type="text" style={{ width: "100%" }} value={microscopeName} onChange={this.handleMicroscopeNameChange} ></input>
+                        </div>
+                        <div>
+                            <label>Validation Tier</label>
+                        </div>
+                        <div>
+                            <DropdownButton id="search-item-type-selector" onSelect={this.onChangeValidationTier.bind(this)}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use .bind() (react/jsx-no-bind)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)